contracts/chain-adapters/l2/WithdrawalAdapter.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.19;
+
+import "../../libraries/CircleCCTPAdapter.sol";
```

Comment:
Nick's recent commit used explicit imports for CCTP; do we want to replicate that in this file https://github.com/across-protocol/contracts/commit/d197e7d543777bad8725d82ae859122544f5b6f2

Reply:
Yeah. I'll start doing that from now on. https://github.com/across-protocol/contracts/pull/605/commits/d3f624bff2d3b263be3a310a5c9ef36a19075bc0